providers/hubspot/subscriptionEvent.go

```diff
@@ -9,24 +9,40 @@ import (
 	"fmt"
 	"strconv"
 	"strings"
+	"time"
 
 	"github.com/amp-labs/connectors/common"
 )
 
+/*
 type SubscriptionEvent struct {
```

Comment:
can you remove this commented out struct? otherwise it'll be tempting for others to revise the code to use this struct in the future, if you wanted to document what the message looks like, then maybe you can provide a sample message in a comment block.